### DISCUSSION TITLE
pyinstaller.exe does not get created in Python\Python312\Scripts

### DISCUSSION THREAD

**Discussion by benn1993** [pyinstaller/pyinstaller#8753] [Help]
pip install pyinstaller seemed to be successful, also it seems like all files are in 
Lib\site-packages

However no exe is appearing in 
Lib\site-packages

therefore 

pyinstaller --version 

results in an unknown command.

I am using 
Python 3.12.0

**Comment by bwoodsend** [comment]
The exe lives in `Scripts`, not `site-packages`. Presumably you haven't added that directory to `PATH`.
